samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingTopicSubscriptionClient/Program.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BasicSendReceiveUsingTopicSubscriptionClient
+{
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Primitives;
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        const string ServiceBusConnectionString = "{Service Bus connection string}";
+        const string TopicName = "{Topic Name}";
+        const string SubscriptionName = "{Subscription Name}";
+        static ITopicClient topicClient;
+        static ISubscriptionClient subscriptionClient;
```

Comment:
I thought about that but then without introducing the concept of rules, i was not sure if creating two subscriptions with default filters was a good idea. When we add the rules sample to demonstrate AddRule/GetRule, we can actually cover the two subscription idea better with that.